src/install/mongodb/default-install-linux.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
+headingText: Configure the MongoDB integration
+---
+
+1. Change directory to the integrations configuration folder by running:
+
+```shell
+cd /etc/newrelic-infra/integrations.d
+```
+
+2. Copy the sample configuration file by running:
+
+```shell
+sudo cp mongodb3-config.yml.sample mongodb3-config.yml
+```
+
+3. Edit the `mongodb3-config.yml` file. The only two required fields are `mongodb_cluster_name` and `mongodb_uri`. This example config file collects all metrics:
```

Comment:
I think it's good practice to explain what required fields do and why they're required (also i'm assuming this will be swapped out, but this is a mongo file th at's also showing up in the Oracle doc)